builtin/providers/aws/resource_aws_vpc_peering_connection.go

```diff
@@ -139,23 +142,23 @@ func resourceAwsVPCPeeringRead(d *schema.ResourceData, meta interface{}) error {
 	// When the VPC Peering Connection is pending acceptance,
 	// the details about accepter and/or requester peering
 	// options would not be included in the response.
-	if pc.AccepterVpcInfo != nil && pc.AccepterVpcInfo.PeeringOptions != nil {
+	if pc.AccepterVpcInfo.PeeringOptions != nil {
```

Comment:
There is no need for this extra check. Above, we set items that are being read from the stricture too which are essential e.g. `	d.Set("peer_vpc_id", pc.AccepterVpcInfo.VpcId)`. Should this fail it would be long before we even get to peering options.